### PR TITLE
fix: icon tests, disabled button with href

### PR DESCRIPTION
**Pull request by demetrius-mp**
forgot to check the tests after my last contribution

### CODE REVIEW

File: src/Button.svelte [hunk 1]
```diff
@@ -38,7 +38,7 @@
   <a
     {...$$restProps}
     class={classes}
-    {disabled}
+    class:disabled
```
Comment (bestguy) [inline src/Button.svelte:41]: `disabled` is a real prop on HTML buttons, why the switch to a class name?

**Review by bestguy — CHANGES_REQUESTED** [review]
Thanks @demetrius-mp , one question on this.

File: src/Button.svelte (hunk 1 above)
Comment (demetrius-mp) [reply src/Button.svelte:41]: not sure what you mean
i removed the disabled attribute of the anchor tag, as it does not have a disabled attribute.

Comment (bestguy) [reply src/Button.svelte:41]: Ah thanks for the clarification, missed that

**Pull request closed without merging**